main/webservices/registration.soap.php

```diff
@@ -2014,7 +2013,6 @@ function WSEditUserWithPicture($params)
     $user
         ->setEmail($email)
         ->setStatus($status)
-        ->setOfficialCode($official_code)
```

Comment:
You should not remove this one. Rather, you should put it in a conditional, like $status, above.